src/components/Form/Form.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react'
+//import { Field, reduxForm } from 'redux-form'
+import DatePicker from 'react-datepicker'
+import moment from 'moment'
+
+import 'react-datepicker/dist/react-datepicker.css'
+import './Form.css'
+
+class Form extends Component {
+
+  constructor(props){
+    super(props)
+    this.state = {
+      date: moment()
+    }
+    this.dateChanged = this.dateChanged.bind(this)
+  }
+
+  dateChanged(date){
+    this.setState({
+      date: date
+    })
+  }
+
+  render() {
+    return (
+      <div className='date'>
+        <DatePicker selected={this.state.date}
+                    onChange={this.dateChanged}
+                    dateFormat="YYYY/MM/DD"
+        />
+      </div>
+    )
+  }
+}
+
+export default Form
```

Comment:
Ensure that you have this setting in your IntelliJ IDE
![selection_513](https://user-images.githubusercontent.com/1978331/35770426-9c6756c4-094d-11e8-96cc-8622a91767d8.png)

Reply:
agreed with @spolischook 
@AlenaMandrika check this